modules/api/pkg/api/v2/types.go

```diff
@@ -208,6 +208,12 @@ type SeedSettings struct {
 	Metering kubermaticv1.MeteringConfiguration `json:"metering"`
 	// the Seed level seed dns overwrite
 	SeedDNSOverwrite string `json:"seedDNSOverwrite,omitempty"`
+	// KubeLB settings
+	KubeLB *KubeLBSeedSettingsAPI `json:"kubeLB,omitempty"`
+}
+
+type KubeLBSeedSettingsAPI struct {
+	EnableForAllDatacenters bool `json:"enableForAllDatacenters,omitempty"`
 }
```

Comment:
Note for reviewer:

Using a new dedicated struct for API here instead of the struct from KKP since we want to limit the fields that are returned through API.